spec/models/package_manager/nu_get_spec.rb

```diff
@@ -1,36 +1,63 @@
 # frozen_string_literal: true
-require 'rails_helper'
+
+require "rails_helper"
 
 describe PackageManager::NuGet do
-  let(:project) { create(:project, name: 'foo', platform: described_class.formatted_name) }
+  let(:project) { create(:project, name: "foo", platform: described_class.formatted_name) }
 
   it 'has formatted name of "NuGet"' do
-    expect(described_class.formatted_name).to eq('NuGet')
+    expect(described_class.formatted_name).to eq("NuGet")
   end
 
-  describe '#package_link' do
-    it 'returns a link to project website' do
+  describe "#package_link" do
+    it "returns a link to project website" do
       expect(described_class.package_link(project)).to eq("https://www.nuget.org/packages/foo/")
     end
 
-    it 'handles version' do
-      expect(described_class.package_link(project, '2.0.0')).to eq("https://www.nuget.org/packages/foo/2.0.0")
+    it "handles version" do
+      expect(described_class.package_link(project, "2.0.0")).to eq("https://www.nuget.org/packages/foo/2.0.0")
     end
   end
 
-  describe 'download_url' do
-    it 'returns a link to project tarball' do
-      expect(described_class.download_url('foo', '1.0.0')).to eq("https://www.nuget.org/api/v2/package/foo/1.0.0")
+  describe "download_url" do
+    it "returns a link to project tarball" do
+      expect(described_class.download_url("foo", "1.0.0")).to eq("https://www.nuget.org/api/v2/package/foo/1.0.0")
     end
   end
 
-  describe '#install_instructions' do
-    it 'returns a command to install the project' do
+  describe "#install_instructions" do
+    it "returns a command to install the project" do
       expect(described_class.install_instructions(project)).to eq("Install-Package foo")
     end
 
-    it 'handles version' do
-      expect(described_class.install_instructions(project, '2.0.0')).to eq("Install-Package foo -Version 2.0.0")
+    it "handles version" do
+      expect(described_class.install_instructions(project, "2.0.0")).to eq("Install-Package foo -Version 2.0.0")
+    end
+  end
+
+  describe "with release data" do
+    def stub_releases(releases_data = {})
+      allow(described_class)
+        .to receive(:get_releases)
+        .and_return(releases_data)
+    end
+
+    # TODO: license urls are deprecated, but for now we'll fallback and include them as the license
```

Comment:
What do people tend to put in there? Is it like a link to MIT?

Reply:
It's generally been ms shortlinks like http://go.microsoft.com/fwlink/?LinkId=329770